### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by daviwil**
This will drop tomorrow morning (1/19) after we finalize some last-minute details.

### CODE REVIEW

**Comment by daviwil** [comment]
@rkeithhill @kapilmb can you read through the changelog and make sure I've covered everything correctly?

**Comment by daviwil** [comment]
Pasting some screenshots for the changelog:

image

image

**Comment by rkeithhill** [comment]
LGTM.  This is a significant update!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,139 @@
...
+debugging PowerShell modules, while we work on a true interactive console experience using
+VS Code's Terminal interface.
+
+#### Other fixes and improvements
+
+- Added a new `Open Examples Folder` command to easily open the extension's
+  example script folder.
+- Added a new setting `powershell.developer.powerShellExeIsWindowsDevBuild`
+  which, when true, indicates that the `powerShellExePath` points to a Windows
+  PowerShell development build.
```
Comment (kapilmb) [inline CHANGELOG.md]: I would suggest an addition here:

Added support for providing symbol outline for psd1 files.

Comment (daviwil) [inline CHANGELOG.md]: Yes!  I forgot about that one.

Comment (kapilmb) [inline CHANGELOG.md]: Yeah, that was a last minute check-in.

**Comment by daviwil** [comment]
Another screenshot:

image

**Pull request merged**